### PR TITLE
feat: merge pisp thirdparty endpoint seeds from pisp/master

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
     "Sam Kummary <sam@modusbox.com>",
     "Steven Oderayi <steven.oderayi@modusbox.com>",
     "Valentin Genev <valentin.genev@modusbox.com>",
-    "Shashikant Hirugade <shashikant.hirugade@modusbox.com>"
+    "Shashikant Hirugade <shashikant.hirugade@modusbox.com>",
+    "Paweł Marzec <pawel.marzec@modusbox.com>",
+    "Kevin Leyow <kevin.leyow@modusbox.com>"
```
Comment (eoln) [inline package.json:20]: what about bumping the version?

**Review by eoln — CHANGES_REQUESTED** [review]
lgtm, the package version needs to be bumped manually...

File: package.json (hunk 1 above)
Comment (kleyow) [reply package.json:20]: I was hoping to have https://github.com/mojaloop/central-ledger/pull/818 to do it for me but now that I think of it, it wouldn't catch this `feat` in the bump.

done and ty.

**Review by eoln — APPROVED** [review]
lgtm

**Pull request merged**